plugins/inputs/burrow/README.md

```diff
@@ -0,0 +1,95 @@
+# Telegraf Plugin: Burrow
+
+Collect Kafka topic, consumer and partition status
+via [Burrow](https://github.com/linkedin/Burrow) HTTP [API](https://github.com/linkedin/Burrow/wiki/HTTP-Endpoint).
+
+Supported Burrow version: `1.x`
+
+### Configuration
+
+```
+  ## Burrow API endpoints in format "schema://host:port".
+  ## Default is "http://localhost:8000".
+  servers = ["http://localhost:8000"]
+
+  ## Override Burrow API prefix.
+  ## Useful when Burrow is behind reverse-proxy.
+  # api_prefix = "/v3/kafka"
+
+  ## Maximum time to receive response.
+  # response_timeout = "5s"
+
+  ## Limit per-server concurrent connections.
+  ## Useful in case of large number of topics or consumer groups.
+  # concurrent_connections = 20
+
+  ## Filter out clusters by providing list of glob patterns.
+  ## Default is no filtering.
+  # clusters = []
```

Comment:
Rename to `clusters_exclude` so that in the future you can add `cluster_include`, or add them both now and you can use `filters.NewIncludeExcludeFilter(`.  Also applies to `groups` and `topics`.

Reply:
Thanks, totally missed that `filter` package is already present and very useful. I did `include`/`exclude` separation and replaced `glob` with `filter`.